.github/workflows/create.yml

```diff
@@ -27,6 +27,16 @@ jobs:
           echo "## Commit results" >> $GITHUB_STEP_SUMMARY
           echo "✅ Passed" >> $GITHUB_STEP_SUMMARY
 
+      - name: "Remove `rename.sh` and `create.yml`"
+        run: |
+          rm -rf "./.github/scripts/rename.sh"
+          rm -rf "./.github/workflows/create.yml"
```

Comment:
There's actually no need to pass the `-r` flag here.

`-r` means recursive, and it's needed only for directories.